mmedit/models/components/layers/feature_shift.py

```diff
@@ -0,0 +1,31 @@
+import torch
+import torch.nn as nn
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class FeatureShift(nn.Conv2d):
```

Comment:
As discussed in private, it is a feature normalization. So "FeatureNormalize" maybe more suitable.

Reply:
ImageNormalize